examples/OneOpenAir/OneOpenAir.ino

```diff
@@ -843,15 +840,42 @@ static void configUpdateHandle() {
 
   if (ag->isOne()) {
     if (configuration.isLedBarBrightnessChanged()) {
-      ag->ledBar.setBrighness(configuration.getLedBarBrightness());
-      Serial.println("Set 'LedBarBrightness' brightness: " +
+      if (configuration.getLedBarBrightness() == 0) {
+        ag->ledBar.setEnable(false);
+      } else {
+        if (configuration.getLedBarMode() != LedBarMode::LedBarModeOff) {
+          ag->ledBar.setEnable(true);
+        }
+        ag->ledBar.setBrighness(configuration.getLedBarBrightness());
+      }
+      ag->ledBar.show();
+      Serial.println("Set 'ledBarBrightness' brightness: " +
                      String(configuration.getLedBarBrightness()));
     }
+
+    if (configuration.isLedBarModeChanged()) {
+      Serial.println("Set 'ledBarMode' " + String(configuration.getLedBarMode()));
+      Serial.println("Get 'ledBarBrightness' " + String(configuration.getLedBarBrightness()));
+      if (configuration.getLedBarBrightness() == 0) {
```

Comment:
I think the code is redundant to what we have on line 843 onwards?

Reply:
Yes, I make it for test. We can remove it.